client/src/socket.io.js

```diff
@@ -10,14 +10,21 @@ export default (store) => {
   socket.on('connect', () => {
     // store.dispatch(setSessionId(socket.id));
   });
+
+  socket.on(actions.UPDATE_GAME_LIST, (data) => {
+    store.dispatch(actionCreators.updateGameList(data.gameList));
+  });
 };
 
 // eslint-disable-next-line no-unused-vars
 export const socketMiddleware = store => next => (action) => {
+  // let messages = store.getState().messages;
+
   const result = next(action);
-  // if (socket && action.type === ADD_MESSAGE) {
-  //   let messages = store.getState().messages;
-  //   socket.emit('message', action.message);
-  // }
+  if (!socket) { return result; }
+  const { type, ...data } = action;
+  if (type === actions.CREATE_GAME) {
+    socket.emit(actions.CREATE_GAME, { ...data });
```

Comment:
Your other comments got me thinking about this and the actions emitted on the backend. These follow a similar format as the redux actions. The redux actions right now are like 
` { type: 'MY_TYPE', dataItem1: 'value1', dataItem2: 'value2' } ` and the sockets are
```js
const type = 'MY_TYPE'
const data = { dataItem1: 'value1', dataItem2: 'value2' }
socket.emit(type, data)
``` 
i believe due to how socketIO works (if . you find other docs let me know).

We could use action creators here and on the backend. i'm not going to refactor now, as I want to merge this so we can work off it, but we can think about doing that at some point in another PR